CloverSDK.podspec

```diff
@@ -0,0 +1,55 @@
+Pod::Spec.new do |s|
+  s.name         = "CloverSDK"
+  s.version      = "0.3.0"
+  s.summary      = "Clover SDK"
+  s.homepage     = "https://www.clover.com/features/integrations"
+  s.license      = { :type => "MIT", :file => "LICENSE" }
```

Comment:
Yeah, the gradle license plugin automatically populates those values with the current date and name supplied via the build.gradle.  I believe the gradle task is licenseFormatMain